setup.cfg

```diff
@@ -1,5 +0,0 @@
-[versioneer]
```

Comment:
This file is used by `versioneer` command. It should not be removed if user will want to upgrade versioneer related files.